examples_test.go

```diff
@@ -28,7 +29,7 @@ func ExampleDB_Save() {
 
 	// Open takes an optional list of options as the last argument.
 	// AutoIncrement will auto-increment integer IDs without existing values.
```

Comment:
I see that this AutoIncrement comment is left here, but the commit message and change in example seems to indicate that the feature is being removed. Will auto-increment no longer be available? Why? Is this comment correct or should it be removed?

Reply:
@PaluMacil It should be removed yeah, good catch 👍